src/scrapers/leumi-card.js

```diff
@@ -162,6 +162,15 @@ async function getAccountNumber(page, cardIndex) {
 async function getTransactionsForSection(page, cardIndex, sectionIndex) {
   const cardSections = await getCardSections(page, cardIndex);
   const txnsRows = await cardSections[sectionIndex].$$('.jobs_regular');
+  const expandedBusinessesNamesHeaders = await cardSections[sectionIndex].$$('.openedJob .bisName');
+  let expandedBusinessesNames = await Promise.all(expandedBusinessesNamesHeaders.map(
+    header => page.evaluate(y => y.innerText, header),
+  ),
+  expandedBusinessesNamesHeaders);
```

Comment:
@orzarchi can you please remove the line break and place this in the previous line?

Reply:
I was struggling with the linting rules for a few minutes.
May I suggest implementing prettier and eslint autofix as a pre-commit hook?
I can add it to my PR or create a new one.